### DISCUSSION TITLE
Dynamic Structure Factor Calculation

### DISCUSSION THREAD

**Discussion by amruthesht** [glotzerlab/freud#1212] [General]
Hello All,

Are there any plans to incorporate dynamic structure factor (self-intermediate structure factor in particular) calculation functionality into freud in the near future.

If not, are there any efficient open-source packages that anyone here has used previously and would suggest using?

Thank you!

**Comment by tommy-waltmann** [comment]
Hi @amruthesht ,

There are no plans to add dynamic structure factors right now, however we would gladly accept a pull request which added this feature. I will leave it to others to make recommendations for other packages to use, as I am not aware of any right now.
